pycheribuild/boot_cheribsd/__init__.py

```diff
@@ -72,7 +72,7 @@
                                                                 CompilationTargets.CHERIBSD_MORELLO_PURECAP,
                                                                 )}
 
-AUTOBOOT_PROMPT = re.compile(r"(H|, h)it \[Enter] to boot ")
+AUTOBOOT_PROMPT = re.compile(r"((H|, h)it \[Enter] to boot |\[Space] to pause)")
```

Comment:
I should maybe add a comment as the 'Hit' version is what the boot loader uses when there is no lua or forth, ', hit' is the old string in lua, and '[Space]' is in the new lua string (and also I think in the old lua string).